test/tests/operations/ByteStringLiteral.mjs

```diff
@@ -0,0 +1,215 @@
+/**
+ * Literal tests.
+ *
+ * @author edouard hinard []
+ * @copyright Crown Copyright 2018
+ * @license Apache-2.0
+ */
+import TestRegister from "../../TestRegister";
```

Comment:
I think this import has moved now & please add file extension to the import: `TestRegister.mjs`